### PR TITLE
Add player method to set playback start time ahead of downloading chunks

### PR DESCRIPTION
**Pull request by sanbornhnewyyz**

### CODE REVIEW

File: lib/player/http_video_source.js [hunk 1]
```diff
@@ -168,6 +171,13 @@ shaka.player.HttpVideoSource.prototype.enableTextTrack = function(enabled) {
 
 
 /** @override */
+shaka.player.HttpVideoSource.prototype.setPlaybackStartTime =
+    function(startTime) {
+  this.playbackStartTime_ = startTime;
```
Comment (joeyparrish) [inline lib/player/http_video_source.js]: Either this should be a no-op for HttpVideoSource, or this value should be used to set video.currentTime in attach().

Comment (sanbornhnewyyz) [inline lib/player/http_video_source.js]: Okay, I've changed to a no-op.

**Comment by joeyparrish** [comment]
I'm going to try out the new CI system on you.

**Comment by sanbornhnewyyz** [comment]
Woohoo!

**Comment by joeyparrish** [comment]
Okay, I'm turning it over to Tim to finish review.

**Comment by tdrews** [comment]
Thanks for taking care of this!

**Pull request merged**